docs/guides/zone-replication.md

```diff
@@ -11,6 +11,8 @@ It is completely possible that all the replicas for the given data are held with
 
 For this reason, Cortex optionally supports zone-aware replication. When zone-aware replication is **enabled**, replicas for the given data are guaranteed to span across different availability zones. This requires Cortex cluster to run at least in a number of zones equal to the configured replication factor.
 
+Reads from a zone-aware replication enabled Cortex Cluster can withstand zone failures as long as there are more than replication factor / 2 zones available without any failing instances.
```

Comment:
> as long as there are more than replication factor / 2 zones available without any failing instances

This is not true. Think about 5 zones and RF=3. You're asserting reads will succeed as far as there are 2 healthy zones, but actually you need 4 healthy zones.

```suggestion
Reads from a zone-aware replication enabled Cortex Cluster can withstand zone failures as long as there are no more than `floor(replication factor / 2)` zones with failing instances.
```